### PR TITLE
autoselect mobile ucr report

### PR DESCRIPTION
**Pull request by dannyroberts**
Gets rid of the extra select screen for Mobile UCR. This is all it took!

### CODE REVIEW

**Comment by dannyroberts** [comment]
@czue / @proteusvacuum @orangejenny

**Comment by orangejenny** [comment]
nice, good once tests pass

fyi @amstone326 your autoselect feature is making mobile UCR better

**Comment by amstone326** [comment]
Nice :) Thanks for the tag

**Comment by czue** [comment]
whaaat?! amazing! :star:

**Comment by dannyroberts** [comment]
(oops, forgot to update tests)

File: corehq/apps/app_manager/tests/test_report_config.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def test_filter_entry(self):
             <instance id="reports" src="jr://fixture/commcare:reports"/>
             <session>
               <datum id="report_filter_a98c812873986df34fd1b4ceb45e6164ae9cc664_computed_owner_name_40cc88a0_1" nodeset="instance('reports')/reports/report[@id='a98c812873986df34fd1b4ceb45e6164ae9cc664']/filters/filter[@field='computed_owner_name_40cc88a0_1']/option" value="./@value" detail-select="reports.a98c812873986df34fd1b4ceb45e6164ae9cc664.filter.computed_owner_name_40cc88a0_1"/>
-              <datum id="report_id_a98c812873986df34fd1b4ceb45e6164ae9cc664" nodeset="instance('reports')/reports/report[@id='a98c812873986df34fd1b4ceb45e6164ae9cc664']" value="./@id" detail-select="reports.a98c812873986df34fd1b4ceb45e6164ae9cc664.select" detail-confirm="reports.a98c812873986df34fd1b4ceb45e6164ae9cc664.summary"/>
+              <datum id="report_id_a98c812873986df34fd1b4ceb45e6164ae9cc664" nodeset="instance('reports')/reports/report[@id='a98c812873986df34fd1b4ceb45e6164ae9cc664']" value="./@id" detail-select="reports.a98c812873986df34fd1b4ceb45e6164ae9cc664.select" detail-confirm="reports.a98c812873986df34fd1b4ceb45e6164ae9cc664.summary" autoselect="true"/>
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_report_config.py]: E501 line too long (349 > 115 characters)

**Comment by czue** [comment]
:boom:

**Comment by snopoke** [comment]
more :candy:

**Pull request merged**